rest/data/commit_queue.go

```diff
@@ -318,6 +318,28 @@ func EnqueuePRToCommitQueue(ctx context.Context, env evergreen.Environment, sc C
 		return nil, errors.New("PR contains no base branch label")
 	}
 
+	if !thirdparty.IsUnblockedGithubStatus(pr.GetMergeableState()) {
```

Comment:
I think we should apply this change (GetMergeableState, getWhatever) to the other calls in this function as well.